install_config/master_node_configuration.adoc

```diff
@@ -1203,8 +1203,21 @@ openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/openp
 
 The advanced audit feature provides several improvements over the
 xref:master-node-config-audit-config[basic audit functionality], including
-fine-grained events filtering and multiple output back ends. The following table
-contains additional options you can use.
+fine-grained events filtering and multiple output back ends. 
+
+To setup the advanced audit feature please add the following entries at the `openshift_master_audit_config` variable
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/oscp-audit/-oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/security/adv-audit.yaml","logFormat":"json"}
+----
+
+[IMPORTANT]
+====
+The policy file *_/etc/security/adv-audit.yaml_* *must* be availabe on on *all* master nodes 
```

Comment:
s/\*must\* be availabe on on \*all\* master nodes/must be available on each master node.

Where do you get the /etc/security/adv-audit.yaml file? It's not defined here, and I don't see it anywhere else in the collection.

Reply:
done